scripts/cut_release_branch.py

```diff
@@ -35,6 +35,20 @@
 
 
 def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
+    r"""Checks that the new version name matches the expected pattern.
+
+    Args:
+        arg: str. The new version name.
+        pattern: RegularExpression. The pattern that release version should
+        match. Default is re.compile(re'\d\.\d\.\d').
```

Comment:
Please indent the continuation lines (ex- 43) by 4 more spaces.

Reply:
Done, commit(https://github.com/oppia/oppia/pull/5728/commits/e0068ec14f34752ce9738c65973ec00bc74d600c)